docs/runtime_CLI.md

```diff
@@ -1,85 +1,200 @@
+TODO: counter_reset
+TODO: help
+TODO: load_new_config_file
+TODO: mc_dump
+TODO: mc_mgrp_create
+TODO: mc_mgrp_destroy
+TODO: mc_node_associate
+TODO: mc_node_create
+TODO: mc_node_destroy
+TODO: mc_node_dissociate
+TODO: mc_node_update
+TODO: mc_set_lag_membership
+TODO: meter_array_set_rates
+TODO: meter_get_rates
+```
+
+### meter_set_rates
+
+The syntax for the CLI is provided below.
+
+```
+RuntimeCmd: help meter_set_rates
+Configure rates for a meter: meter_set_rates <name> <index> <rate_1>:<burst_1> <rate_2>:<burst_2> ...
+```
+
+Meter behavior is specified in [RFC 2697](https://tools.ietf.org/html/rfc2697)
+and [RFC 2698](https://tools.ietf.org/html/rfc2698).
+
+The user enters the meter rate in bytes/microsecond and burst_size in bytes.  If
+the meter type is packets, the rate is entered in packets/microsecond and
+burst_size is the number of packets.
+
+
+```
+TODO: port_add
+TODO: port_remove
+TODO: register_read
+TODO: register_reset
+TODO: register_write
+TODO: reset_state
+TODO: serialize_state
+TODO: set_crc16_parameters
+TODO: set_crc32_parameters
+TODO: shell
+```
+
+### show_actions
+
+No parameters.  For every action in the currently loaded P4 program, shows the
+name and list of parameters.  For each parameter, the name and width in bits is
+shown.
+
+
+### show_ports
+
+No parameters.  Shows the port numbers, the interface names they are associated
+with, and their status (e.g. up or down).
+
+
+### show_tables
+
+No parameters.  For every table in the currently loaded P4 program, shows the
+name, implementation (often `None`, but can be an action profile or action
+selector for tables created with those options), and a list of table search key
+fields, giving for each such field its name, match kind (e.g. `exact`, `lpm`,
+`ternary`, `range), and width in bits.
+
+
+```
+TODO: swap_configs
+TODO: switch_info
+```
+
+
+### table_add
+
+```
+RuntimeCmd: help table_add
+Add entry to a match table: table_add <table name> <action name> <match fields> => <action parameters> [priority]
+```
+
+You can find a list of all table and action names via the commands `show_tables`
+and `show_actions`.  The default name created if you do not use a `@name`
+annotation to specify one in your P4 program is often hierarchical, starting
+with the name of a control that the table or action is defined within.
+
+Match fields must appear in the same order as the table search key fields are
+specified in the P4 program, without names.  The output of `show_tables` also
+shows the names, match kinds (`ternary`, `range`, `lpm`, or `exact`), and bit
+widths for every table, after the `mk=` string.
+
+Numeric values can be specified in decimal, or hexadecimal prefixed with `0x`.
+As a special case, 32-bit values can be specified like IPv4 addresses, in dotted
+decimal, e.g. `10.1.2.3`, and 48-bit values can be specified like Ethernet MAC
+addresses, with each byte in two hexadecimal digits, with bytes separated by
+`:`, e.g. `00:12:34:56:78:9a`.
```

Comment:
v6 addresses can also be specified using standard IETF representation

Reply:
I added a note about that, with some examples and a link to the relevant RFC specifying the syntax